sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/src/Services/ServiceConfigParser.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+
+namespace Microsoft.Azure.WebJobs.Extensions.WebPubSub
+{
+    internal class ServiceConfigParser
+    {
+        private static char[] _valueSeparator = new char[] { '=' };
+
+        public Uri Endpoint { get; }
+
+        public string AccessKey { get; }
+
+        public string Version { get; }
+
+        public int Port { get; }
+
+        public ServiceConfigParser(string connectionString)
+        {
+            var settings = ParseConnectionString(connectionString);
+
+            Endpoint = settings.ContainsKey("endpoint") ?
+                new Uri(settings["endpoint"]) :
+                throw new ArgumentException(nameof(Endpoint));
+            AccessKey = settings.ContainsKey("accesskey") ?
+                settings["accesskey"] :
+                throw new ArgumentException(nameof(AccessKey));
+
+            Version = settings.ContainsKey("version") ? settings["version"] : null;
+            Port = settings.ContainsKey("port") ? int.Parse(settings["port"], CultureInfo.InvariantCulture) : 80;
+        }
+
+        private static Dictionary<string, string> ParseConnectionString(string connectionString)
+        {
+            if (string.IsNullOrEmpty(connectionString))
+            {
+                throw new ArgumentException("Web PubSub Service connection string is empty");
+            }
+
+            var setting = new Dictionary<string, string>();
+            var items = connectionString.Split(';');
+
+            try
+            {
+                setting = items.Where(x => x.Length > 0).ToDictionary(x => x.Split(_valueSeparator, 2)[0], y => y.Split(_valueSeparator, 2)[1], StringComparer.InvariantCultureIgnoreCase);
```

Comment:
Can use this overload https://docs.microsoft.com/en-us/dotnet/api/system.string.split?view=net-5.0#System_String_Split_System_Char_System_Int32_System_StringSplitOptions_

Reply:
It's not supported in netstandard 2.0